src/olympia/users/forms.py

```diff
@@ -198,6 +195,10 @@ class Meta:
         model = UserProfile
         exclude = USER_EDIT_EXCLUDE_FIELDS
 
+    def clean_email(self):
+        # TODO(django 1.10): Change the field to disabled=True and remove this.
```

Comment:
It's actually in 1.9, I updated the comment. The [`disabled` attribute](https://docs.djangoproject.com/en/1.10/ref/forms/fields/#disabled) to a field will set the `disabled` attribute of the input and ignore any value sent in the form.

> The disabled boolean argument, when set to True, disables a form field using the disabled HTML attribute so that it won’t be editable by users. Even if a user tampers with the field’s value submitted to the server, it will be ignored in favor of the value from the form’s initial data.